src/js/utils/aria.js

```diff
@@ -0,0 +1,18 @@
+define([
+
+], function() {
+
+    const addLabel = function(element, ariaLabel) {
+        if (!element || !ariaLabel) {
+            return;
+        }
+
+        element.setAttribute('aria-label', ariaLabel);
+        element.setAttribute('role', 'button');
+        element.setAttribute('tabindex', '0');
+    };
+
+    return {
+        addLabel: addLabel
+    };
```

Comment:
Just return the function. Waste of an object and `. addLabel` can't be minified.